packages/venia-concept/src/components/Checkout/flow.js

```diff
@@ -47,16 +46,14 @@ class Flow extends Component {
         isPaymentMethodReady: bool,
         isShippingInformationReady: bool,
         isShippingMethodReady: bool,
-        paymentMethod: string,
-        paymentTitle: string,
+        paymentData: object,
```

Comment:
`availablePaymentMethods` and `paymentMethod` are no longer used, and the UI needs more than just the `paymentTitle`.  That information is stored in `paymentData`.

Reply:
In that case maybe update to use `shape` like some of the other props?